packages/pages/src/App.js

```diff
@@ -92,6 +101,31 @@ export default provideState({
         await this.effects.refreshEntries();
       }
     },
+    async submitUpdates(effects, event) {
+      event.preventDefault();
+      const response = await fetch("/api/", {
+        method: "post",
+        body: format.request(0, "updateEntry", {
+          id: this.state.id,
+          name: this.state.name,
+          content: this.state.content,
+        }),
+      });
+      const parsed = parse(await response.text());
+      if (parsed.type === "error") {
+        console.error(parsed.error);
+      } else if (parsed.type === "response") {
+        this.state.name = "";
+        this.state.content = "";
+        this.state.id = "";
+        await this.effects.refreshEntries();
+      }
+    },
```

Comment:
`createEntry` and `submitUpdates` share a lot of code/behavior, don't think it should be possible to unite them?